### PR TITLE
Trento v2.3.0 change#10

### PR DESCRIPTION
**Pull request by abravosuse**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

**Review by abravosuse — COMMENTED** [review]
Remark about change #9 is obsolete, as this has become a branch and a PR for change #10.

**Review by tomschr — APPROVED** [review]
@abravosuse Thank you, looks fine! :+1:

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -2701,12 +2703,13 @@ trento-server-rabbitmq-0</screen>
             <entry>2.0.0</entry>
             <entry>2.1.0</entry>
             <entry>2.2.0</entry>
+            <entry>2.3.0</entry>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2706]:
```suggestion
            <entry>2.3.1</entry>
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -2767,6 +2775,18 @@ trento-server-rabbitmq-0</screen>
             <entry><phrase role="color:suse-jungle-green">✓</phrase></entry>
             <entry><phrase role="color:suse-jungle-green">✓</phrase></entry>
             <entry><phrase role="color:suse-jungle-green">✓</phrase></entry>
+            <entry></entry>
+          </row>
+                    <row>
+            <!-- <entry/> -->
+            <entry>2.3.0</entry>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2782]:
```suggestion
            <entry>2.3.1</entry>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -2801,7 +2821,44 @@ trento-server-rabbitmq-0</screen>
     <!-- HINT: The list should be sorted by versions, having the most recent version at the
                first item.
     -->
-    <variablelist>
+       <variablelist>
+      <varlistentry xml:id="trento-version-230">
+        <term>Version 2.3.0 (released on 2024/MM/DD)</term>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2826]:
```suggestion
        <term>Version 2.3.1 (released on 2024/06/18)</term>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -2801,7 +2821,44 @@ trento-server-rabbitmq-0</screen>
...
+        <listitem>
+          <para> Consisting of Helm chart 2.3.0, Web component 2.3.0, checks engine
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2828]:
```suggestion
          <para> Consisting of Helm chart 2.3.1, Web component 2.3.1, checks engine
```

**Review by abravosuse — COMMENTED** [review]
@tomschr the new version is finally out. I've made a few little changes because, in the end, the version of the helm chart and the web component is 2.3.1 instead of 2.3.0. The rest remains the same.

**Pull request merged**